snaps/how-to/use-keyring-api/create-account-snap.md

```diff
@@ -213,8 +213,8 @@ Notify MetaMask when the following events take place, using the `emitSnapKeyring
 
 ### 6. Expose the Keyring API
 
-Create a handler function using the `onKeyringRequest` export to expose the Keyring API methods to
-MetaMask and your dapp:
+Create a handler function using the `onKeyringRequest` entry point to expose the Keyring API methods
```

Comment:
I am wondering if the right wording is "Create an entry point using the `onKeyringRequest` handler to expose..."

Reply:
How about "Create an `onKeyringRequest` entry point handler to expose the Keyring API methods"?